app/src/main/java/com/ybsystem/tweethub/activities/PostActivity.java

```diff
@@ -0,0 +1,155 @@
+package com.ybsystem.tweethub.activities;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import androidx.fragment.app.FragmentTransaction;
+
+import com.ybsystem.tweethub.R;
+import com.ybsystem.tweethub.application.TweetHubApp;
+import com.ybsystem.tweethub.fragments.fragment.PostFragment;
+import com.ybsystem.tweethub.libs.eventbus.PostEvent;
+import com.ybsystem.tweethub.models.entities.EntityArray;
+import com.ybsystem.tweethub.utils.DialogUtils;
+import com.ybsystem.tweethub.utils.GlideUtils;
+import com.ybsystem.tweethub.utils.KeyboardUtils;
+import com.ybsystem.tweethub.utils.ToastUtils;
+
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.ybsystem.tweethub.models.enums.ImageOption.NONE;
+
+public class PostActivity extends ActivityBase {
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make mImageUris a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)